### PR TITLE
fix: remove price sort in collection

### LINKED ISSUE #2632: Filter function seems to be broken in Explore > Collections
### What happened?

The filtering function seems to be acting weird, it sometimes shows the same nfts and it definitely doesn't load the last interacted collections, based on "latest sales" on landing page.

### Please reproduce in steps

Go to kodadot.xyz
Click on Explore>Collections>Filter by

### Expected Behavior

/

### What browsers are you seeing the problem on?

Mozilla/Chrome

### Are you logged in?

yes

### At which address did you encounter bug?

kodadot.xyz

### Screenshots

https://user-images.githubusercontent.com/92786519/159474570-9c32dcfa-f4a3-416c-8560-7e27c3289339.mp4

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2632
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Review by vikiival — CHANGES_REQUESTED** [review]
Code looks oki
I would just rename `defaultSort` to something more meaningful

File: components/rmrk/Collection/List/CollectionList.vue [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="collections">
     <Loader :value="isLoading" />
-    <Search v-bind.sync="searchQuery" @resetPage="currentValue = 1">
+    <Search
+      v-bind.sync="searchQuery"
+      @resetPage="currentValue = 1"
+      :defaultSort="collectionSortConfigure">
```
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/rmrk/Collection/List/CollectionList.vue]: done!

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
updated #2641

**Pull request merged**